lncfg/db.go

```diff
@@ -57,6 +57,24 @@ func (db *DB) Validate() error {
 	return nil
 }
 
+func (db *DB) InitEtcd(ctx context.Context, dbPath string) error {
```

Comment:
Missing a `godoc` method. Given this is `etcd` specific (and also just for testing?), does it need to be exported publicly?

Reply:
Added `godoc`. Renamed to `Init` since it may become more generic over time. We could hide it behind `GetBackends` but then `GetBackends` should be called quite early in the startup process, before leader election which is more confusion imho. wdyt?